circleci/integration-test

```diff
@@ -93,9 +93,13 @@ download_test_artifacts(){
   for workflowID in $(cat integration-tests.out | jq -r '.testWorkflowIds[]');
   do
     S3_BUCKET_URL="s3://frontend-to-end-tester-files-dev/${workflowID}/artifacts.zip"
+    S3_BUCKET="frontend-to-end-tester-files-dev"
+    S3_ARTIFACTS_KEY="${workflowID}/artifacts.zip"
```

Comment:
Nit: Suggest changing `S3_BUCKET_URL` to be the composite of these: `S3_BUCKET_URL="s3://${S3_BUCKET}/${S3_ARTIFACTS_KEY}"`